diff -up ./CMakeLists.txt.orig ./CMakeLists.txt
--- ./CMakeLists.txt.orig	2017-12-31 15:36:15.381194454 +0200
+++ ./CMakeLists.txt	2017-12-31 15:36:48.277948721 +0200
@@ -183,8 +183,8 @@ endif()
 
 
 set(prefix ${CMAKE_INSTALL_PREFIX})
-set(exec_prefix ${prefix}/bin)
-set(libdir ${prefix}/lib)
+set(exec_prefix ${prefix}/libexec)
+set(libdir ${prefix}/lib${LIB_SUFFIX})
 set(includedir ${prefix}/include)
 set(ORTP_PKGCONFIG_VERSION "${ORTP_VERSION}")
 set(ORTPDEPS_LIBS )
